java/main/com/google/privacy/differentialprivacy/SecureNoiseMath.java

```diff
@@ -85,4 +85,18 @@ public static double roundToMultipleOfPowerOfTwo(double x, double granularity) {
       return x;
     }
   }
+
+  /** Computes the closest double value that is larger than or equal to the provided long value. */
+  public static double nextLargerDouble(long n) {
```

Comment:
I believe you don't need to add this (when Karim will merge his version). We improved documentation a bit already, so better to keep his version